docs/secure-connections/sauce-connect/installation.md

```diff
@@ -154,25 +154,38 @@ The launch of Sauce Connect 4.7.1 makes it the officially supported version of t
    </td>
   </tr>
   <tr>
-   <td rowspan="2" >4.7
+   <td rowspan="2" >4.8
    </td>
-   <td>4.7.1
+   <td>4.8.0
    </td>
    <td>
     <a href="#download-latest-version">See latest version</a>
```

Comment:
This changes impacts the "Downloading Sauce Connect Proxy" section in this same topic (see below), and also impacts other Sauce Connect topics that refer or link to version 4.7.1, including: **/secure-connections/sauce-connect/quickstart/#start-tunnel** and **/secure-connections/sauce-connect/setup-configuration/basic-setup/#basic-setup-with-a-test-script**

<img width="839" alt="Screen Shot 2022-05-11 at 12 13 10 PM" src="https://user-images.githubusercontent.com/15352964/167897837-68b65f07-0595-449e-9546-830883b320c0.png">

Reply:
I've been told that these links and checksums auto populate with each new release, and that they didn't need to be updated.

Reply:
But you're right - I didn't see the code snippet, and that will need to be updated.

Reply:
I take that first comment back - apparently, the auto populate feature was removed from this page.